satviz-serial/src/main/java/edu/kit/satviz/serial/ClauseUpdateSerializer.java

```diff
@@ -4,15 +4,24 @@
 import java.io.IOException;
 import java.io.OutputStream;
 
+/**
+ * A {@link Serializer} for {@code ClauseUpdate}s, i.e. the combination of a {@code Clause} and
+ * a {@code ClauseUpdate.Type}.<br>
+ * Uses one byte for the type and the rest as specified by {@link ClauseSerializer}.
+ */
 public class ClauseUpdateSerializer extends Serializer<ClauseUpdate> {
+
+  private static final ClauseSerializer clauseSerializer = new ClauseSerializer();
+
   @Override
   public void serialize(ClauseUpdate clauseUpdate, OutputStream out)
       throws IOException, SerializationException {
-
+    out.write(clauseUpdate.type().ordinal());
```

Comment:
Hier wollen wir uns an die binary DRAT Spezifikation halten: ADD hat byte 97 ('a'), DELETE hat byte 100 ('d').
Das könnten wir vielleicht direkt im ClauseUpdate.Type enum als Attribut speichern.

Reply:
Dann könnte man sich auch diesen mäßig schönen Umweg über .values() in ClauseUpdateSerialBuilder sparen